### PR TITLE
10555 division qb read only

### PR DESCRIPTION
**Pull request by DevneyHamilton**
This adds a 'qb_read_only' field on division, set in Accounting Settings. It defaults to 'true.' If 'true' then interest calculations will not run on any loans, and admins will not see the 'add transaction' button on the transactions page for a loan.

### CODE REVIEW

**Comment by DevneyHamilton** [comment]
**User facing changes**
On the Accounting settings form, there is a new checkbox for this field. 
Screen Shot 2020-03-24 at 11 39 37 AM

@emitche - this has not had a UX pass. Please feel free to make any improvements if you want to on this branch (or we can revisit in a later sprint). 

On the transactions page for a loan, if accounting settings are set to true, the user will not see the 'add transactions' button
Screen Shot 2020-03-23 at 7 20 20 PM


**Behind the scenes changes**
Interest calculations should not be created when loans are updated.

File: spec/features/admin/accounting/transaction_flow_spec.rb [hunk 1]
```diff
@@ -41,7 +42,19 @@
       scenario 'create new transaction button is hidden' do
         visit "/admin/loans/#{loan.id}/transactions"
         expect(page).to have_content('Transactions are read-only')
-        expect(page).not_to have_button('Add Transaction')
+        # expect "Add Transaction" to not be available
+        expect(page).not_to have_selector('.btn[data-action="new-transaction"]')
+      end
+    end
+
+    context "loan's qb_division has are qb_read-only on" do
```
Comment (emitche) [inline spec/features/admin/accounting/transaction_flow_spec.rb]: wording tweak

**Review by emitche — APPROVED** [review]
Code review:
* Code looks good.
* There is a test that could use renaming.

For UX, I can do a UX pass on this.

File: spec/features/admin/accounting/transaction_flow_spec.rb (hunk 1 above)
Comment (DevneyHamilton) [reply spec/features/admin/accounting/transaction_flow_spec.rb]: thanks for catching this :)

**Comment by emitche** [comment]
UX changes are in #751.

**Comment by emitche** [comment]
QA:

When the read-only setting is checked and accounting settings are saved:
* The Add Transaction button is not visible on a loan
* The Update Transactions button is visible on loan index - **BUG**

When the read-only setting is unchecked and accounting settings are saved:
* The Add Transaction button is visible on a loan
* The Update Transactions button is visible on loan index

**Comment by emitche** [comment]
It was an easy fix. Updating code soon.

**Comment by emitche** [comment]
QA Update:

This works as expected.

When the read-only setting is checked and accounting settings are saved:
* The Add Transaction button is not visible on a loan
* The Update Transactions button is not visible on loan index

When the read-only setting is unchecked and accounting settings are saved:
* The Add Transaction button is visible on a loan
* The Update Transactions button is visible on loan index

**Comment by emitche** [comment]
Functionality to always see the Update Transactions button has been restored. That was not a bug.

**Pull request merged**